bundles/org.openhab.binding.heliosventilation/src/main/java/org/openhab/binding/heliosventilation/internal/HeliosVentilationConfiguration.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.heliosventilation.internal;
+
+import org.eclipse.jdt.annotation.NonNull;
+
+/**
+ * The {@link HeliosVentilationConfiguration} class contains fields mapping thing configuration parameters.
+ *
+ * @author Raphael Mack - Initial contribution
+ */
+public class HeliosVentilationConfiguration {
+
+    /**
+     * Port name for a serial connection to RS485 bus. Valid values are e.g. COM1 for Windows and /dev/ttyS0 or
+     * /dev/ttyUSB0 for Linux.
+     */
+    public @NonNull String serialPort = "";
```

Comment:
You could also set `@NonNullByDefault` on class-level. But I'm not sure if that is a good idea. `""` seems to invalid anyway, so there is no benefit over null?

Reply:
I agree, but as SerialPortManager.getIdentifier requires a non-null parameter I'd like to keep the @NonNull and "" init. Even with another init value it can be invalid which leads to a null return in getIdentifier. "/dev/ttyUSB0" is as invalid on Windows as "COM7" on Linux. Anyways the user has to give a value here.

Reply:
I would suggest to use `@NonNullByDefault` on class-level. The primitive `int` can't be null anyway.